src/data-acess/user.transfer.ts

```diff
@@ -0,0 +1,17 @@
+import { Sequelize } from "sequelize";
+import { UserModel } from "../interfaces/user.types";
+import { sequelizeUserModel } from "../models/user.model";
+
+export const createConnection = (url: string): Sequelize => {
+    return new Sequelize(url);
+}
```

Comment:
A connection should be created once on the applications startup. That's not a user module responsibility.

Reply:
it's running once after run application

Reply:
Yeah, but it's not a part of user entity